XXXX-template.md

```diff
@@ -0,0 +1,55 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- Proposal PR: solana-foundation/SImD#0000
+- Feature Gate: solana-foundation/SImD#0000
+- Tracking Issue: solana-labs/SImD#0000
+- Implementation Status: Implemented/In Progress/Planned
```

Comment:
We might want to use YAML front matter here. This allows parsing proposal metadata if we ever want to deploy and index on a website. See SIMD-0002 for an example.

Reply:
Updated to use front matter. Good catch